pandas/formats/format.py

```diff
@@ -2138,6 +2016,31 @@ def formatter(value):
 
         return formatter
 
+    def _get_formatted_data(self):
+        """
+        Returns formatted float values. This is different from
```

Comment:
why are you making this a separate function ? the point was to eliminate this kind of thing;